src/modeler/AutoRest.Swagger/Validation/CollectionObjectPropertiesNamingValidation.cs

```diff
@@ -26,6 +26,11 @@ public class CollectionObjectPropertiesNamingValidation : TypedRule<Dictionary<s
         /// </summary>
         public override ValidationCategory ValidationCategory => ValidationCategory.RPCViolation;
 
+        /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
is this rule serviceimpacting? I'm thinking it's mostly around the modeling of pageable in the swagger, thoughts?